modules/users/tests/server/user.server.routes.tests.js

```diff
@@ -742,7 +742,7 @@ describe('User CRUD tests', function () {
               }
 
               // Call the assertion callback
-              userInfoRes.body.message.should.equal('Username already exists');
+              userInfoRes.body.message.toLowerCase().should.containEql('username already exists');
```

Comment:
Please add a comment for this line and L797 explaining what string MongoDB 2.x and what string MongoDB 3.x expect so it's clear why this change was made.